package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "egeria-ui",
+  "version": "0.0.1",
+  "description": "Egeria UI application",
+  "author": "Egeria contributors",
+  "license": "Apache-2.0",
+  "repository": "odpi/egeria-ui",
+  "vaadin": {
+    "disableUsageStatistics": true
+  },
+  "scripts": {
+    "serve": "hs -c0 ./build/dev/ -P http://localhost:9000",
```

Comment:
https?

Reply:
this command is only used for development, -P http://localhost:9000 is a flag for proxying all the missing request to the odpi/egeria-api-mocks

it connects to the mocks who don't really need to be https, but i think we can achieve that in a future PRs and i think it's actually a good idea to start https directly from development phases